src/routers/productExports.ts

```diff
@@ -1,24 +1,171 @@
+ * @apiParam {String} productType The type of product.
+ * @apiParam {Number} amountOfProduct The amount of the product in it's given units.
+ *                    The product's units can be checked via /product
+ *
+ * @apiError (400) BadRequest Missing or invalid fields, ...
+ * @apiError (403) Forbidden Current user type does not have sufficient privileges.
+ *
+ * @apiSuccess (201) {String} Success Successfully created new product export entry
+ * @apiSuccessExample Success-Response:
+   {
+     "uuid": "9f11efb5-d5b4-4853-aa88-ee10c2940c9f",
+   }
+ */
 router.post('/', async (req, res) => {
-  // TODO
-  res.status(StatusCode.CREATED).send('Successfully created new product export');
-}, authorized(UserType.ADMIN));
+  const createReq: ProdExportReq = {
+    uuid: req.body.uuid,
+    transportId: req.body.transportId,
+    datetime: req.body.datetime,
+    productType: req.body.productType,
+    amountOfProduct: req.body.amountOfProduct,
+  };
+
+  try {
+    const uuid = await ProdExportsService.createProdExportInDb(createReq);
+    res.status(StatusCode.CREATED).send({ uuid });
+  } catch (e) {
+    if (e.message === ProdExportsService.unhandledErrorMsg) {
+      res.status(StatusCode.INTERNAL_SERVER_ERROR).send();
+    } else {
+      res.status(StatusCode.BAD_REQUEST).send('BadRequest ' + e.message);
+    }
+  }
+}, authorized(UserType.ADMIN, UserType.TRADER));
 
+/**
+ * @api {put} /productexports/:uuid
+ * @apiName UpdateProductExport
+ * @apiGroup ProductExports
+ * @apiVersion 0.0.1
+ * @apiDescription Updates a product export entry.
+ *              Any fields not given will not be updated.
+ *              Extra non existent fields will not be ignored.
+ *              Returns the product export's actual data on error or success.
+ *
+ * @apiParam {String} uuid Specify the UUID of the product export to update.
+ * @apiParam [String] transportId Identifier for mode of transport, max 255 characters.
+ * @apiParam [String] datetime The time the export occured.
+ * @apiParam [String] productType The type of product.
+ * @apiParam [Number] amountOfProduct The amount of the product in it's given units.
+ *                    The product's units can be checked via /product
+ *
+ * @apiError (403) Forbidden Current user type does not have sufficient privileges.
+ * @apiError (404) NotFound Product export not found
+ *
+ * @apiSuccess (200) {String} Successfully updated product export entry
+  {
+    "uuid": "3321269e-9b8b-432f-a668-b65c206235b0",
+    "transportId": "A3C-X23",
+    "datetime": "2018-01-23 04:05:20",
+    "lastModified": "2018-01-23 04:06:21",
+    "productType": "corn",
+    "amountOfProduct": 5.2
+  }
+*/
 router.put('/:uuid', async (req, res) => {
-  // TODO
-  res.status(StatusCode.OK).send('Successfully updated product export');
-}, authorized(UserType.ADMIN));
+  const updateReq: ProdExportUpdateReq = {
+    uuid: req.params.uuid,
+
+    transportId: req.body.transportId,
+    datetime: req.body.datetime,
```

Comment:
they shouldn't be able to change datetime? but maybe the client should be generating the last modified timestamp?